mod/message.php

```diff
@@ -420,7 +420,7 @@ function render_messages(array $msg, string $t): string
 	$tpl = Renderer::getMarkupTemplate($t);
 	$rslt = '';
 
-	$myprofile = DI::baseUrl() . '/profile/' . $a->getLoggedInUserNickname();
+	$myprofile = DI::baseUrl() . '/profile/' . DI::userSession()->getLocalUserNickname();
```

Comment:
It looks like the `$a = DI::app()` call in this function can be removed as this seems it was the only references to it.